src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -2001,7 +2001,16 @@ type Stop implements Node & PlaceInterface {
   "Identifier of the platform, usually a number. This value is only present for stops that are part of a station"
   platformCode: String
   "Routes which pass through this stop"
-  routes: [Route!]
+  routes(
+    """
+    Only include routes whose pattern operates on at least one service date specified by this filter.
```

Comment:
Maybe we shouldn't mention anything about patterns here, either just say that the route is operational on at least one of the days or it has at least one departure on at least one of the days.

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/6003/commits/86b48dbc46d95995b7a3e6330cef5f9602d87952